man/Makefile.am

```diff
@@ -1,5 +1,11 @@
 AUTOMAKE_OPTIONS = gnu
 
```

Comment:
it's worth checking if **make install** does the right thing. Would it be possible to have a .rst.8: rule instead ?